### PR TITLE
Add syllabus meeting minutes for 02-03-2022

### PR DESCRIPTION
**Pull request by Dedekind561**
## Description

Adds notes for syllabus meeting 02-03-2022

## Who needs to know about this?

@CodeYourFuture/syllabus-team 

## Rendered Pages

-----
[View rendered docs/contributing/minutes.md](https://github.com/CodeYourFuture/syllabus/blob/minutes/03-02-2022/docs/contributing/minutes.md)

### CODE REVIEW

File: docs/contributing/minutes.md [hunk 1]
```diff
@@ -4,6 +4,73 @@ title: Team Minutes
...
+- Getting alums to attend syllabus meetings - really valued input as they've just done the courses
+- Make teams more accountable on Slack - have bots that remind about meetings in public channels ?
+- Someone could join JS2 to suppor them with getting through some tasks
+- Do we need syllabus team folks to join the module teams to get them going ( time limited )
+
+### Welcoming Sanyia
+
+- We welcome to Sanyia to the Syllabus Team
+- Software Engineer at Deliveroo Started at CYF 2 years ago
+- Some node module changes proposals ( Barath / Sanyia ) - Use of wordle in Lesson 3 of the Node module
```
Comment (Dedekind561) [inline docs/contributing/minutes.md:49]: @sansaid Please could you add a link to your node module proposals - I think I saw a google doc in the meeting

File: docs/contributing/minutes.md [hunk 2]
```diff
@@ -4,6 +4,73 @@ title: Team Minutes
...
+### Celebrating fundamentals
+
+- Woop! 🎉 Go Fundamentals - now published!
+- Fundamentals training starting on 12th Match for S African volunteers
+- Fundamentals starts in April with South Africa Class 2
+
+### Plan 2022
+
+- Improve student tracking
+- Further delegation to module teams
```
Comment (Dedekind561) [inline docs/contributing/minutes.md]: @gregdyke Think you went through a list of post-its but I may have missed off one or 2 if you want add them in here

Comment (SallyMcGrath) [inline docs/contributing/minutes.md]:
```suggestion
- Further empowerment of module teams
```

File: docs/contributing/minutes.md [hunk 3]
```diff
@@ -4,6 +4,73 @@ title: Team Minutes
...
+- Cross module stuff
```
Comment (SallyMcGrath) [inline docs/contributing/minutes.md]:
```suggestion
- Cross module threads: code review, TDD, problem solving
```

File: docs/contributing/minutes.md [hunk 4]
```diff
@@ -4,6 +4,73 @@ title: Team Minutes
...
+-
+- Iterate on module template for Code Your Future
+- Succession planning: learning how to hand things over in a sustainable way 🤔
+
+## Module teams & expectations
+
+- Work needed to ensure that module teams are meeting regularly
+- Getting alums to attend syllabus meetings - really valued input as they've just done the courses
+- Make teams more accountable on Slack - have bots that remind about meetings in public channels ?
+- Someone could join JS2 to suppor them with getting through some tasks
```
Comment (SallyMcGrath) [inline docs/contributing/minutes.md]:
```suggestion
- Someone could join JS2 to support them with getting through some tasks
```

File: docs/contributing/minutes.md [hunk 5]
```diff
@@ -4,6 +4,73 @@ title: Team Minutes
...
+- Do we need syllabus team folks to join the module teams to get them going ( time limited )
+
+### Welcoming Sanyia
+
+- We welcome to Sanyia to the Syllabus Team
```
Comment (SallyMcGrath) [inline docs/contributing/minutes.md]:
```suggestion
- We welcome Sanyia to the Syllabus Team
```

File: docs/contributing/minutes.md [hunk 6]
```diff
@@ -4,6 +4,73 @@ title: Team Minutes
...
+
+### Sprint planning
+
+- [Syllabus project board](https://github.com/CodeYourFuture/syllabus/projects/1) to undergo a lot of trimming down and breaking out into module boards
+- Create a wordle exercise for Node ( Sanyia )
+- [Blockly project board](https://github.com/CodeYourFuture/fundamentals-blockly/projects/1) updated with new tasks
+
+### S Africa mentor training
+
+- Greg to lead with support from Mitch
```
Comment (SallyMcGrath) [inline docs/contributing/minutes.md]:
```suggestion
- Greg to lead with support from Mitch 12 March
```

File: docs/contributing/minutes.md [hunk 7]
```diff
@@ -4,6 +4,73 @@ title: Team Minutes
...
+- Mention pedagogic techniques in remot environment: no slides, cold calling, screen sharing, live code demos etc ...
```
Comment (SallyMcGrath) [inline docs/contributing/minutes.md]:
```suggestion
- Mention pedagogic techniques in remote environment: no slides, cold calling, screen sharing, live code demos etc ...
```

**Review by SallyMcGrath — CHANGES_REQUESTED** [review]

**Comment by SallyMcGrath** [comment]
@all-contributors please add @Dedekind561 for review, tests and code

**Pull request merged**